### ISSUE TITLE
Markup up of dates and editorial insertions for the year

### ISSUE THREAD

**Issue by MikeJ1971** [ilrt/Beyond2022TeiTemplate#23]
The markup of the regnal dates look like this:

`<date type="regnal" when="1420-07-25">25 July [1420] 8 Henry V</date>`

I thought the @type value of ‘regal’ could be added for clarification since there might be other date formats.

Do we need or want to add an editorial insertion for the year? The year could be rendered in the presentation layer.

So, rather than saying 

`25 July [1420] 8 Henry V`

We tag it as:

`<date type="regnal" when="1420-07-25">25 July 8 Henry V</date>.`

The style sheets can add the year based on the @when attribute: ‘25 July 8 Henry V [1420]’, or ‘25 July [1420] 8 Henry V’.

**Update**

Here is an example with the regnal date:

```
Wishing to be informed of the arrears of his fees from <date
               when="1420-04-26">26 April last</date> to <date when="1420-04-09">9 July last</date>
```

**Comment by munnellg** [comment]
Good idea to add the type. We can handle regnal years in the database. What we'll do on the back end is take the content of the "when" attribute and use that to look up the appropriate regnal year.
